src/annotator/test/guest-test.js

```diff
@@ -427,6 +395,34 @@ describe('Guest', () => {
         );
       });
     });
+
+    describe('on "loadAnnotations" event', () => {
+      it('anchors annotations', () => {
+        const guest = createGuest();
+        const ann1 = { id: 1, $tag: 'tag1' };
+        const ann2 = { id: 2, $tag: 'tag2' };
+        sandbox.stub(guest, 'anchor');
```

Comment:
I think this follows the pattern of the previous tests. This will do for now, although stubbing an internal method is not ideal, as what you want to check really are the externally visible effects of anchoring. One way to do this is to listen for the `anchorsChanged` event that is emitted when anchoring completes.